WordPress/Sniffs/VIP/SuppressFiltersSniff.php

```diff
@@ -8,61 +8,203 @@
  */
 
 /**
- * Suppress Filters need to be 'true' when getting posts using get_posts.
+ * Support_filters must be false when using get_posts(), wp_get_recent_posts(), get_children().
```

Comment:
I've replaced that first doc comment because it was wrong. if `suppress_filters` value is false then it's fine otherwise in case it didn't pass or it is true then the warning will occur. so the doc string can be.

> Flag calling get_posts(), wp_get_recent_posts() and get_children() without suppress_filters=>false.

OR

> Checks for suppress_filters=>false being supplied in get_posts(), wp_get_recent_posts() and get_children().

Please, correct.